src/main/java/org/candlepin/util/Util.java

```diff
@@ -620,4 +621,37 @@ else if (temporalAccessor instanceof LocalDateTime) {
     public static String encodeUrl(String text) {
         return URLEncoder.encode(text, StandardCharsets.UTF_8);
     }
+
+    /**
+     * Returns the list items which got re-ordered in modified list.
```

Comment:
Let's explain in more detail that this method will only find the items that were re-ordered compared to each other (not as a side-effect of adding/removing items)